### PR TITLE
Updated vagrant script to fix bcrypt missing error and npm version

### PR DESCRIPTION
**Pull request by yugensoft**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
https://habitica.com/#/options/groups/guilds/68d4a01e-db97-4786-8ee3-05d612c5af6f

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Updated npm version to match http://habitica.wikia.com/wiki/Setting_up_Habitica_Locally
Added npm-pre-gyp dependency necessary for bcrypt to install

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID: 6365790b-b0ea-4da6-bbe3-f5a217330ad2

### CODE REVIEW

**Comment by TheHollidayInn** [comment]
Looks good to me. @paglias does this seem right to you?

File: vagrant_scripts/install_node.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ nvm use
 nvm alias default current
 
 echo Update npm...
-npm install -g npm@3
+npm install -g npm@4.5.0
```
Comment (paglias) [inline vagrant_scripts/install_node.sh]: can this be just npm@4?

**Review by paglias — CHANGES_REQUESTED** [review]

File: vagrant_scripts/install_node.sh (hunk 1 above)
Comment (yugensoft) [reply vagrant_scripts/install_node.sh]: ok sure ill change it

Comment (yugensoft) [reply vagrant_scripts/install_node.sh]: ok its done

**Comment by Alys** [comment]
I'm merging this because the current vagrant setup is wrong and this is at the very least an improvement, even if further changes are needed later.

@yugensoft Thank you! I'm awarding you your second tier for this PR and the investigative work needed to get vagrant working.

As your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required.

**Pull request merged**